src/components/teamMember.js

```diff
@@ -0,0 +1,138 @@
+import React from "react"
+import { Box, Text, Image, Flex } from "rebass"
+import { keyframes } from "@emotion/core"
+
+import memberPlaceholder from "../images/member-placeholder.jpg"
+
+const float = keyframes`
+  to {
+    transform: translateY(-0.3rem);
+  }
+`
+
+const scaleIn = keyframes`
+  from {
+    opacity: 1;
+    transform: scale(1);
+  }
+
+  to {
+    opacity: 0;
+    transform: scale(0);
+  }
+`
+
+const animationScaleIn = `${scaleIn} 300ms 0 both`
+
+const scaleOut = keyframes`
+  from {
+    opacity: 0;
+    transform: scale(0);
+  }
+
+  to {
+    opacity: 1;
+    transform: scale(1);
+  }
+`
+
+const animationScaleOut = `${scaleOut} 300ms both`
+
+/**
+ * Utility to emulate the responsive array syntax for non-style props.
+ * @param {*} prop - The prop to be made responsive.
+ * @param {string} propValueFn - Callback whose return value will be
+ *     used to determine the value of a style for each breakpoint.
+ *     If prop is an array, it will receive the corresponding value
+ *     at each iteration. Otherwise, it will be called with just the
+ *     value of prop.
+ */
+const responsiveProp = (prop, propValueFn) => {
+  if (Array.isArray(prop)) {
+    return prop.map(p => propValueFn(p))
+  }
+
+  return propValueFn(prop)
+}
+
+const TeamMember = ({
+  isHidden,
+  isHighlighted,
+  member: { photo, name, role, website },
+  onMouseEnter: handleMouseEnter,
+  ...props
+}) => (
+  <Box
+    as="a"
+    href={website}
+    target="_blank"
+    rel="noopener noreferrer"
+    onMouseEnter={handleMouseEnter}
```

Comment:
I think`onMouseEnter` will not work on mobile. Add touch events or remove any hover action on mobile.

Reply:
True. 🤔 I'd like to keep it though so at least if the user taps on someone's name, the effect will still show up.